Makefile

```diff
@@ -130,14 +130,41 @@ oc-clean: oc-down
 	$(PREFIX) rm -rf $(OC_DATA_DIR)
 
 user:
-	$(PYTHON) $(PYDIR)/manage.py createsuperuser --settings=config.settings.local
+	http post localhost:8080/auth/users/create/ \
+	    username=user password=password
+	$(PYTHON) $(PYDIR)/manage.py drf_create_token user --settings=config.settings.local
+
+superuser:
+	-$(PYTHON) $(PYDIR)/manage.py createsuperuser \
+	    --settings=config.settings.local \
+	    --username=admin --email=admin@admin.com --noinput
```

Comment:
Let's use a "fake" domain like `example.com`, and not admin, we wouldn't want to accidentally send emails to this domain.

Reply:
Good point. Done.